package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@zeit/git-hooks",
+  "name": "@vercel/git-hooks",
   "version": "0.2.1",
   "description": "No nonsense Git hook management",
-  "repository": "zeit/git-hooks",
-  "author": "Josh Junon <junon@zeit.co>",
+  "repository": "vercel/git-hooks",
+  "author": "Josh Junon <josh@junon.me>",
```

Comment:
```suggestion
```

Reply:
Could you explain this change, please? Do I not maintain this anymore?

Reply:
Sorry, I will be honest I didn't know you created this entirely. I saw an old ZEIT email so I figured it needed to be removed. Happy to keep you as the author!